atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryDbTimeLockFactory.java

```diff
@@ -51,7 +51,7 @@ public KeyValueService createRawKeyValueService(
     public ManagedTimestampService createManagedTimestampService(
             KeyValueService rawKvs, DbTimestampCreationSetting dbTimestampCreationSetting, boolean initializeAsync) {
         return services.computeIfAbsent(
-                dbTimestampCreationSetting.timestampSeries(), (unused) -> new InMemoryTimestampService());
+                dbTimestampCreationSetting.timestampSeries(), unused -> new InMemoryTimestampService());
     }
```

Comment:
may have to underscore it? but if you've run the build  this is fine.

Reply:
I _think_ that if you call it "unused" it doesn't mind, but we'll see!